sos/report/plugins/block.py

```diff
@@ -59,11 +59,13 @@ def setup(self):
         self.add_blockdev_cmd(cmds, blacklist='ram.*')
 
         lsblk = self.collect_cmd_output("lsblk -f -a -l")
-        # for LUKS devices, collect cryptsetup luksDump
+        # for LUKS devs, pull cryptsetup luksDump, clevis, and /etc/crypttab.
```

Comment:
Maybe worth explaining (on an extra comment line) that we need to collect `/etc/crypttab` when *any* LUKS device exists, hence `add_copy_spec` called for every such device. E.g.:

        # for LUKS devices, collect cryptsetup, luksDump and clevis
        # if a LUKS device exists, ensure /etc/crypttab is collected

That will also resolve the concise language due to line length :).